dojo/forms.py

```diff
@@ -2776,6 +2777,37 @@ class Meta:
         exclude = ["template"]
 
 
+class NotificationsWebhookForm(forms.ModelForm):
+    class Meta:
+        model = Notification_Webhooks
+        exclude = []
+
+    def __init__(self, *args, **kwargs):
+        is_superuser = kwargs.pop("is_superuser", False)
+        logger.debug(f"is_superuser: {is_superuser}")
+        super().__init__(*args, **kwargs)
+        self.fields["status"].disabled = True  # TODO: - same for API
+        self.fields["first_error"].disabled = True
+        self.fields["last_error"].disabled = True
+        self.fields["note"].disabled = True
```

Comment:
This sounds good. I will need to move `first_error` and `last_error` to `List` page as some note otherwise, the user is not able to see when an error happened. Right now, the user can see it in the `Edit` form.

Reply:
Main part done: https://github.com/DefectDojo/django-DefectDojo/pull/7311/commits/fe41ee87d7d1d332cf34417802a3396b3101ec2e
Note with a date is still needed.

Reply:
Note added: https://github.com/DefectDojo/django-DefectDojo/pull/7311/commits/5e533b44449b37d4a9737efa87da89ed49ec46e7